README.md

```diff
@@ -155,15 +157,6 @@ export const env = validateEnv({
         return value;
       }
     },
-    NODE_ENV: (value) => {
```

Comment:
I hesitated to remove this part, on one side it shows the kind of custom validator we can create with type inference and on the other side it could be confusing because juste above there is an helper showing we don't need to write this kind of validator.

Reply:
You are right, it's less confusing this way.